src/Utils/sockets.cpp

```diff
@@ -1,16 +1,21 @@
 #include "sockets.h"
 
+#ifndef __ZEPHYR__
 #include <netdb.h>
 #include <netinet/in.h>
 #include <sys/socket.h>
+#endif
 #include <unistd.h>
 
+#ifndef __ZEPHYR__
 #include <csignal>
+#endif
 #include <cstdarg>
 #include <cstdio>
 #include <cstdlib>
 #include <cstring>
 
+#ifndef __ZEPHYR__
```

Comment:
Why can we remove these lines of code in `socket.cpp` without replacing them in Zephyr? Or put differently, why do we need them for the other versions, but not Zephyr?

Reply:
Zephyr just uses the Duplex channel. Because of this the other channels are not really referenced anywhere else, so the code can compile just fine without these definitions. The other channels are disabled because open bot brain doesn't have networking.

Reply:
these headers don't exist on zephyr